pkgs/development/tools/analysis/ida-free/update.sh

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p bash curl jq
+
+set -e
+
+urls=(
+  https://out7.hex-rays.com/files/idafree82_linux.run
+  https://out7.hex-rays.com/files/idafree82_mac.app.zip
+  https://out7.hex-rays.com/files/arm_idafree82_mac.app.zip
+)
```

Comment:
What about version bumps? Will we encounter hash a mismatch here if the upstream is updated?

Reply:
oh that's correct. I might have to parse https://hex-rays.com/ida-free/#download.